.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
     - chmod 777 ~/onFailure.sh
     - chmod 777 ~/onSuccess.sh
 install:
-    - mvn dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+    - mvn clean install dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
 script:
     - mvn -B clean verify cobertura:cobertura coveralls:cobertura -Dorg.apache.maven.user-settings=~/settings-arquillian.xml  -Dorg.apache.maven.global-settings=~/settings-arquillian.xml
```

Comment:
We should change this to install http://docs.codehaus.org/display/MAVENUSER/introduction-to-the-lifecycle